src/slashcommands/ISlashCommand.ts

```diff
@@ -0,0 +1,6 @@
+// just guessing
+export interface ISlashCommand {
+    command: string;
+    pattern: string;
+    function: () => boolean;
```

Comment:
"implemented", see [execution result definition](https://github.com/graywolf336/temporary-rocketlets-ts-definition/pull/6/files#diff-1c36bdf22eb1e8e4cba5ff8d057028a8) and [it's usage](https://github.com/graywolf336/temporary-rocketlets-ts-definition/pull/6/files#diff-97e084cbbf47c714fa7298b553b243b4)